### PR TITLE
javascript-loops | Moses Kim

### PR DESCRIPTION
**Pull request by mosesr-kim**
Screen Shot 2021-05-12 at 5 51 03 PM

### CODE REVIEW

File: exercises/javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers[currentNumber - 1] = currentNumber;
```
Comment (codingcodymiller) [inline exercises/javascript-loops/main.js:13]: You should use `push` instead

File: exercises/javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled[i] = numbers[i] * 2;
```
Comment (codingcodymiller) [inline exercises/javascript-loops/main.js:48]: You should use `push` instead

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good start! A couple quick fixes for you:

**Comment by mosesr-kim** [comment]
Screen Shot 2021-05-13 at 12 56 54 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**